### PR TITLE
Add Ruby 2.6.9 version for CVE-2021-41817, CVE-2021-41819

### PR DESCRIPTION
**Pull request by kamoh**
Adds Ruby 2.6.9 image for these CVEs:

Issues summary:
https://www.ruby-lang.org/en/news/2021/11/24/ruby-2-6-9-released/

CVE-2021-41817: https://www.ruby-lang.org/en/news/2021/11/15/date-parsing-method-regexp-dos-cve-2021-41817/
CVE-2021-41819: https://www.ruby-lang.org/en/news/2021/11/24/cookie-prefix-spoofing-in-cgi-cookie-parse-cve-2021-41819/

### CODE REVIEW

**Comment by rvalenciano** [comment]
Did we run any of the ruby 2.6.X current images with this new image? This can be done locally by building the image in your local machine and using it in the Dockerfile of core for instance. If tests pass and you are able to run the app, and do things with it (like logging in, creating an account, updating the password, hitting the post report create endpoint), we should be good.

We also need to do the same process for ruby 2.7.X as we have other apps running in this ruby version.

File: 2.6.9/jemalloc/buster-slim/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,119 @@
+FROM debian:buster-slim
+MAINTAINER Patrick Tulskie <patrick@ltvco.com>
```
Comment (KevinPedroza) [inline 2.6.9/jemalloc/buster-slim/Dockerfile:2]: Can we update this tag since this is now deprecated?

File: 2.6.9/jemalloc/buster/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,93 @@
+FROM buildpack-deps:buster
+MAINTAINER MinLing Zhao <mzhao@beenverified.com>
```
Comment (KevinPedroza) [inline 2.6.9/jemalloc/buster/Dockerfile:2]: Same comment above

File: 2.6.9/jemalloc/buster-slim/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+	mkdir -p /usr/local/etc; \
+	{ \
+		echo 'install: --no-document'; \
+		echo 'update: --no-document'; \
+	} >> /usr/local/etc/gemrc
+
+ENV LANG C.UTF-8
+ENV RUBY_MAJOR 2.6
+ENV RUBY_VERSION 2.6.9
+ENV RUBY_DOWNLOAD_SHA256 5db187882b7ac34016cd48d7032e197f07e4968f406b0690e20193b9b424841f
```
Comment (KevinPedroza) [inline 2.6.9/jemalloc/buster-slim/Dockerfile:31]: Also, this SHA256 is for [ruby 2.6.6](https://www.ruby-lang.org/en/news/2020/03/31/ruby-2-6-6-released/), not [2.6.9](https://www.ruby-lang.org/en/news/2021/11/24/ruby-2-6-9-released/).

Same in buster

**Review by KevinPedroza — CHANGES_REQUESTED** [review]
Small changes above

**Comment by kamoh** [comment]
Closing in favor of this PR: https://github.com/ltvco/bv-ruby/pull/7

**Pull request closed without merging**